solutions/main.tf

```diff
@@ -303,10 +303,117 @@ resource "aws_instance" "application" {
   vpc_security_group_ids = ["${aws_security_group.peekops_main_app.id}"]
   subnet_id = "${aws_subnet.peekops_main_app_a.id}"
   tags {
-     Name = "peekops_application"
+     Name = "peekops_application_a"
+  }
+  provisioner "local-exec" {
```

Comment:
One tool is in use right now is Puppet Master. Other tool like Terraform, barely know it when I took this test